include/picongpu/particles/ionization/byCollision/ThomasFermi/ThomasFermi_Impl.hpp

```diff
@@ -129,12 +136,15 @@ namespace ionization
              */
             using DensitySolver = typename particleToGrid::CreateDensityOperation<T_SrcSpecies>::type::Solver;
 
-            /** Solver for energy density of the electron species with energy cutoff
+            /** Solver for energy density of the electron species with maximum energy cutoff
              *
              *  @todo Include all electron species with a ForEach<VectorallSpecies,...>
              * instead of just the destination species
              */
-            using EnergyDensitySolver = typename particleToGrid::CreateEnergyDensityCutoffOperation<T_DestSpecies>::type::Solver;
+            using EnergyDensitySolver = typename particleToGrid::CreateEnergyDensityCutoffOperation<
+                T_DestSpecies,
+                CutoffMaxEnergy
```

Comment:
Now the cutoff is more generic and the connection to `ThomasFermi` happens only here.
I still feel that the place/way I chose is maybe not too elegant.